### PR TITLE
Barracuda Web Application Firewall basic support

### PR DESCRIPTION
**Pull request by jayceeemperador**
Applicable to all Barracuda WAF models, change some data on the snmprec file for security purposes. Includes firmware version and serial number detection. 

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: mibs/BARRACUDA-REF-MIB [hunk 1]
```diff
@@ -0,0 +1,22 @@
+BARRACUDA-REF DEFINITIONS ::= BEGIN
```
Comment (PipoCanaja) [inline mibs/BARRACUDA-REF-MIB]: This file should be migrated to mibs/barracuda folder.

File: mibs/BWS-MIB [hunk 2]
```diff
@@ -0,0 +1,3211 @@
+BWS-MIB DEFINITIONS ::= BEGIN
+
```
Comment (PipoCanaja) [inline mibs/BWS-MIB]: This file should be migrated to mibs/barracuda folder.

Comment (jayceeemperador) [reply mibs/BWS-MIB]: Sorry, will do

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-59-release-changelog-december-2019/10558/1

**Pull request merged**